sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -318,6 +320,138 @@ case class DescribeCommand(
 }
 
 /**
+ * A command for users to create a new database.
+ *
+ * It will issue an error message when the database with the same name already exists,
+ * unless 'ifNotExists' is true.
+ * The syntax of using this command in SQL is:
+ * {{{
+ *    CREATE DATABASE|SCHEMA [IF NOT EXISTS] database_name
+ * }}}
+ */
+case class CreateDatabase(
```

Comment:
also, why did you move them to this file? These are DDLs so they belong to `ddl.scala`